config/zones/RO.yaml

```diff
@@ -9,45 +9,70 @@ capacity:
     - datetime: '2023-01-01'
       source: entsoe.eu
       value: 118.0
+    - datetime: '2024-12-12'
```

Comment:
Is this the date the data was published or updated?

We would like the use the date it was updated to prevent any regressions if it's updated in the future.

Reply:
The web page states these are the numbers today -- https://anre.ro/puteri-instalate/